.dockerignore

```diff
@@ -0,0 +1,8 @@
+**/bin
+**/obj
+**/out
+**/.vscode
+**/.vs
+.dotnet
+.Microsoft.DotNet.ImageBuilder
```

Comment:
Not needed

Reply:
It is extremely helpful when running `Get-TagsDocumentation`